format_parser.gemspec

```diff
@@ -42,6 +42,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rspec'
   spec.add_development_dependency 'simplecov'
   spec.add_development_dependency 'webmock'
+  spec.add_development_dependency 'webrick'
```

Comment:
afaik, webrick was extracted as a separate Ruby gem and thus that might be why it was causing issues with the latest Ruby versions.

Reply:
Yeah that's what I found out as well. Really odd this was not erroring earlier as I don't think this has changed that recently 🤔